pandas/core/groupby/base.py

```diff
@@ -28,8 +28,8 @@ def f(x):
 
     def _gotitem(self, key, ndim, subset=None):
         """
-        sub-classes to define
-        return a sliced object
+        Sub-classes to define
+        return a sliced object.
```

Comment:
I think they can go to the same line: `Sub-classes to define. Return a sliced object.`